src/index.tsx

```diff
@@ -1,14 +1,13 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import { createRoot } from "react-dom/client";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
 
-ReactDOM.render(
+createRoot(document.body.appendChild(document.createElement("div"))).render(
```

Comment:
Why are we creating a new div?

wouldnt we want to create a root from document.getElementById('root')?